lib/entities/entity.dart

```diff
@@ -92,4 +99,58 @@ extension TransactionUtils on TransactionBase {
   void addArFsTag() {
     addTag(EntityTag.arFs, '0.11');
   }
+
+  get _platformVersion async {
+    final platform = _platform;
+    final deviceInfoPlugin = DeviceInfoPlugin();
+
+    switch (platform) {
+      case 'Android':
+        final androidDeviceInfo = await deviceInfoPlugin.androidInfo;
+        final androidVersion = androidDeviceInfo.version.release;
+        final sdkVersion = androidDeviceInfo.version.sdkInt;
+
+        // 9 (SDK 28)
```

Comment:
why did you add this comment?

Reply:
I'll update these like this
```suggestion
        // E.g. 9 (SDK 28)
```

Reply:
That's an example only

Reply:
Removes the example comments: d30b549e7c791dc601956e8bb377b47a4456d58e